pkg/odo/cli/storage/create.go

```diff
@@ -87,10 +139,19 @@ func NewCmdStorageCreate(name, fullName string) *cobra.Command {
 		},
 	}
 
+	if experimental.IsExperimentalModeEnabled() {
+		storageCreateCmd.Flags().StringVar(&o.devfilePath, "devfile", "./devfile.yaml", "Path to a devfile.yaml")
+	}
+
+	o.isDevfile = experimental.IsExperimentalModeEnabled() && util.CheckPathExists(o.devfilePath)
```

Comment:
Shouldn't this be the other way round as we move out of experimental mode? I mean, first evaluate `o.isDevfile` and then `if o.isDevfile` then `storageCreateCmd.Flags().StringVar(&o.devfilePath, "devfile", "./devfile.yaml", "Path to a devfile.yaml")`